web/portal/user/cypress/e2e/Recording/Recording.cy.ts

```diff
@@ -0,0 +1,59 @@
+import RecordingCollection from '../../fixtures/Recording/getCollection.json';
+import RecordingItem from '../../fixtures/Recording/getItem.json';
+
+describe('CallForwardSetting', () => {
+  beforeEach(() => {
+    cy.prepareGenericPactInterceptors('User-Recording');
+    cy.before();
+
+    cy.get('svg[data-testid="SettingsVoiceIcon"]').first().click();
+
+    cy.get('table').should('contain', RecordingCollection.body[0].duration);
+  });
+
+  it('View CallForwardSetting', () => {
+    cy.usePactIntercept(
+      {
+        method: 'GET',
+        url: '**api/user/recordings/*',
+        response: RecordingItem.body,
+      },
+      'getRecording-1'
+    );
+
+    cy.usePactIntercept(
+      {
+        method: 'GET',
+        url: '**api/user/recordings/*/recordedfile',
```

Comment:
this could work for general programming in recording files, but here, I'm pretty sure that you know the ID of the recording, so it's a better idea to use the ID of this use case than a mask for every request

Reply:
PD: the id is "id": 2, and I recommend writing the string for the interceptor, as follows:
'getRecordedFile-{ID}'

Reply:
```suggestion
        export const putCompany = () => {
  cy.usePactIntercept(
    {
      method: 'GET',
      url: '**/api/brand/companies/1',
      response: { ...CompaniesItem },
    },
    'getCompanies-1'
  );

  cy.usePactIntercept(
    {
      method: 'PUT',
      url: `**/api/brand/companies/${editCompanies.response.body.id}`,
      response: editCompanies.response,
    },
    'editCompanies'
  );

  cy.get('svg[data-testid="EditIcon"]').first().click();

  testPbx({ ...editCompanies, mode: MODE.EDIT });

  cy.get('header').should('contain', 'Virtual PBXs');

  cy.usePactWait(['editCompanies'])
    .its('response.statusCode')
    .should('eq', 200);
};
```